src/App.tsx

```diff
@@ -2,23 +2,94 @@ import React from 'react';
 import './App.scss';
 
 const goodsFromServer: string[] = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
+  'Sex',
+  'Drugs',
+  'Rock-n-Roll',
+  'Love',
+  'Passion',
+  'Alcohol',
+  'Party',
+  'Adventure',
+  'Laugh',
+  'Money',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selectedGoods: string[],
+};
+
+class App extends React.Component<{}, State> {
+  state = {
+    selectedGoods: [],
+  };
+
+  toggler = (good: string) => {
```

Comment:
toggler for what? Not informative name
```suggestion
  toggleSelectedGoods = (good: string) => {
```